src/docs/asciidoc/user-guide/assertj-core-assertions-guide.adoc

```diff
@@ -641,7 +641,7 @@ This section describes the assertions common to all types.
 [[assertj-core-object-assertions]]
 ==== Object assertions (TODO)
 
-TODO
+TODO(see also https://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#describe-assertion[the old document])
```

Comment:
this is already covered here https://assertj.github.io/doc/#assertj-core-assertion-description